### DISCUSSION TITLE
Incompatible server, update help

### DISCUSSION THREAD

**Discussion by razorsmile** [mbround18/valheim-docker#125] [Q&A] [answered]
Hi, I've got an incompatible server after recent updates.  Here's the lines from the log confirming:

02/17/2021 16:06:52: Peer 76561198010478893 has incompatible version, mine:0.143.5 remote 0.145.6

I did a refresh of the command >docker-compose up -d to refresh before this log report, immediately after the first failed login attempt.  I then used command >docker-compose exec --user steam valheim bash to get into the container to look at the log.  I can access odin, (version 1.1.0) so can I just run the command >odin install to get it to refresh through steamcmd?  Will this disrupt saves, docker volumes or such?

**Comment by rinasek** [comment]
I have the same issue, ran odin install and it did not update anything....

**Comment by mbround18** [comment]
I cant even launch the game with the new update, it keeps crashing. Ill have to get back to you after I figure out why it wont launch. As for the versions, its possible they pushed a new version to the client but did not update the dedicated server tool yet. which is what Odin runs on top of.

**Comment by mbround18** [comment]
If you run into this, i resolved it by running the auto_update.sh script. You can do the same by running 

Change to steam if you did not use `--user steam`
```
gosu steam bash # it should put your in the /home/steam/valheim directory
```

As steam user:
```
pwd # this should return /home/steam/valheim if not use cd /home/steam/valheim
odin install
odin stop
odin start
```
![image](https://user-images.githubusercontent.com/12646562/108249535-0c01fa80-710a-11eb-9007-d298762ec53b.png)

**Comment by rinasek** [comment]
I actually did only odin install again and friends are telling me it works. Will check after meetings. Thanks!

**Comment by JohnKreese** [comment, marked as answer]
You don't actually need Odin, although it probably makes things easier. You can just restart the docker container (backup your world file just in case though).

sudo docker ps -a
sudo docker restart name_of_container
